README.md

```diff
@@ -245,6 +245,131 @@ in the correct format as Protobuf encoded binary data.
 `ProtobufDescriptor` is preferred as you might have to pay a performance penalty
 should you want to use the `BigQuerySchema` instead.
 
+### Batch client
+
+The batch client can be used if you want to upload a big dataset of data to bigquery without any additional cost.
+
+Here follows an example on how you can create such a batch API driven BigQuery client.
+
+```go
+import (
+    "bytes"
+    "context"
+    "encoding/json"
+
+    "github.com/OTA-Insight/bqwriter"
+
+    "cloud.google.com/go/bigquery"
+)
+
+// TODO: use more specific context
+ctx := context.Background()
+
+// create a BQ (stream) writer thread-safe client,
+bqWriter, err := bqwriter.NewStreamer(
+    ctx,
+    "my-gcloud-project",
+    "my-bq-dataset",
+    "my-bq-table",
+    &bqwriter.StreamerConfig{
+        // use 5 background worker threads
+        WorkerCount: 5,
```

Comment:
Is this something you recommend or did you simply copy it over, otherwise I would leave it out if not specific to what you're trying to demonstrating here.